### PR TITLE
Steve/postgres database

### PR DESCRIPTION
**Pull request by MHDGhanemBalhawan**
Changed the api for events to /events/  and added a router for mentors /mentors/.

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -6,12 +6,26 @@ const morgan = require("morgan");
...
+app.use(FLOATERS_BASE_URL, router);
+
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header(
+    "Access-Control-Allow-Headers",
+    "Origin, X-Requested-With, Content-Type, Accept"
+  );
+  next();
+});
```
Comment (anton-kozyr) [inline backend/server.js]: Are these headers still required? I thought having ```proxy``` in ```frontend/package.json``` should do the job.

File: backend/package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "body-parser": "^1.18.3",
+    "cors": "^2.8.4",
```
Comment (anton-kozyr) [inline backend/package.json]: I believe this is not used.

**Review by anton-kozyr — APPROVED** [review]
```cors``` and access control headers can be removed

**Comment by fentonfentonfenton** [comment]
The PR/branch should be called something like `implementing CRUD on RESTful endpoints` - currently it's named the same as your last

File: backend/server.js (hunk 1 above)
Comment (MHDGhanemBalhawan) [reply backend/server.js]: I don't know I'm going to remove them and test it.

File: backend/package.json (hunk 2 above)
Comment (MHDGhanemBalhawan) [reply backend/package.json]: I removed it.

**Review by anton-kozyr — APPROVED** [review]

**Review by dahfool — APPROVED** [review]

**Pull request merged**